.github/workflows/test.yml

```diff
@@ -12,15 +12,9 @@ jobs:
     strategy:
       fail-fast: false
       matrix: # https://docs.djangoproject.com/en/stable/faq/install/#what-python-version-can-i-use-with-django
-        django-version: ["3.2", "4.2", "5.0", "5.1", "5.2rc1"]
-        python-version: ['3.9', '3.10', '3.11', '3.12', '3.13', 'pypy-3.10']
+        django-version: ["4.2", "5.0", "5.1", "5.2rc1"]
+        python-version: ['3.9', '3.10', '3.11', '3.12', '3.13', 'pypy-3.11']
```

Comment:
This fails!  Perhaps leave it for a separate pull request.

Reply:
Values have to be changed in both `tox.ini` and in `test.yml`